### DISCUSSION TITLE
Quire responsive for mobile?

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#483] [Get Help] [answered]
Is there a way to set up the cover image so that it is responsive on mobile? This is what mine currently looks like:

![IMG_0157](https://user-images.githubusercontent.com/74433007/189452539-df33d12b-8642-494b-8280-512fc0d44fa0.PNG)

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @mmpadilla15, as I mention in discussion #480 you should be able to fix the cover issue by overriding some CSS. Do you have the project deployed somewhere where we can take a look at it?

**Reply by mmpadilla15** [reply]
Hi @Erin-Cecele! I deployed it to Netlify so that my team can review it this week. It's at [https://art-scene.cartermuseum.org/](https://art-scene.cartermuseum.org/)

**Reply by mmpadilla15** [reply]
Hi @Erin-Cecele Just wanted to check back in on this one ...

**Reply by geealbers** [reply]
@mmpadilla15 try by adding this in your custom.css file.

```css
.quire-cover__overlay {
  background-size: contain;
  background-color: #231f20;
}
```

This tells browsers to adjust the cover image to be "contained" within the space given. And the background color given matches the background color of the image itself so you can's see the edges as the image adjusts in size.

**Reply by mmpadilla15** [reply]
@geealbers This looks much better!
![IMG_0385](https://user-images.githubusercontent.com/74433007/194443471-7b803d43-09db-4ae1-809c-271f7218f352.PNG)
But I'm wondering if there's a way to eliminate the black space at the top and bottom?

**Reply by geealbers** [reply]
Not without some work restructuring the cover template and writing custom css. I'm afraid the layout just wasn't designed to handle this use case. Though I think it's a good one and something we'll keep in mind for future Quire improvements

**Reply by mmpadilla15** [reply]
@geealbers That's kind of what I figured. But I suspect most users won't be reading this on a phone. But ... If we reconfigure the graphic so that it fits better on a phone, is there a way to use it for mobile devices only, and then the wide graphic for web?

**Reply by geealbers** [reply]
This option would be possible, but similarly, not without reworking the template and adding some css rules.
